test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpath-snapshotter.yaml

```diff
@@ -31,10 +31,13 @@ spec:
       containers:
       - name: csi-snapshotter
         # TODO: replace with official 2.0.0 release when ready
-        image: quay.io/k8scsi/csi-snapshotter:v2.0.0-rc1
+        #image: quay.io/k8scsi/csi-snapshotter:v2.0.0-rc1
+        # Test VolumeSnapshot CRD v1beta1
+        image: quay.io/k8scsi/csi-snapshotter:test-sidecar-splitcontroller #testsnapshotbeta
         args:
+        - "--v=5"
         - "--csi-address=$(ADDRESS)"
-        - "--connection-timeout=15s"
+        - "--leader-election=false"

```

Comment:
Is this because the new default is true? We may want to still keep the default off to make it consistent with the other sidecars.

Reply:
Default is off.  I'll remove this.